apps/example-admin/project.json

```diff
@@ -17,10 +17,8 @@
                 "tsConfig": "apps/example-admin/tsconfig.app.json",
                 "inlineStyleLanguage": "scss",
                 "assets": [
-                    {
-                        "glob": "**/*",
-                        "input": "apps/example-admin/public"
-                    }
```

Comment:
This is also required to be there for future. THis ensures that any files we put inside assets folder will be copied when we build the angular project for deployment.